src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -158,7 +158,7 @@ public int aa
                               ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 1) (Syntax: '1')
                           ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.String, Constant: """") (Syntax: 'BB = """"')
                             Left: 
-                              IPropertyReferenceOperation: System.String <anonymous type: System.Int32 aa, System.String BB, ClassA.SSS CCC>.BB { get; } (OperationKind.PropertyReference, Type: System.String) (Syntax: 'BB')
+                              IPropertyReferenceOperation: System.String? <anonymous type: System.Int32 aa, System.String BB, ClassA.SSS CCC>.BB { get; } (OperationKind.PropertyReference, Type: System.String) (Syntax: 'BB')
```

Comment:
Why is BB showing up as nullable here? Is that just a general fallout of the change that anonymous type members always infer to have a nullable return type irrespective of the arguments?

Reply:
That's a known issue with nullability in anonymous types. We create all member properties as nullable. 
This used to only be "when the feature is turned on" (ie LangVersion = 8), but now it's all the time.

Reply:
Can't tell from the comment if that's a bug or intended RTM behavior.